week-1/implement/max.js

```diff
@@ -0,0 +1,11 @@
+function maxNum(list) {
+  let maximum = -Infinity;
+  for (let x of list) {
+    if (x > maximum && !x.isNaN) {
```

Comment:
Also, x is not really useful you should make all your functions and variables more meaningful.